examples/simple_fullscreen.rs

```diff
@@ -0,0 +1,61 @@
+extern crate winit;
+
+use winit::{ControlFlow, Event, WindowEvent};
```

Comment:
Putting this inside of `main` will prevent warnings from being emitted outside of macOS, which would be nice.

Though, instead of having a standalone `simple_fullscreen` example, it could be cool to offer it as an option on `fullscreen` that's only prompted for on macOS.